### ISSUE TITLE
Error running 'cabal configure'

### ISSUE THREAD

**Issue by chshersh** [phlummox/hup#2]
I have problems using `hup` to upload documentation for [`log-warper`](http://hackage.haskell.org/package/log-warper) package. I run the following command:

```
$ hup docboth -u shersh --password=<my password> --verbose
```

And I see this output:

```
haddock --hyperlinked-source
build dependencies docs
stack haddock --only-dependencies
Haddock index for snapshot packages already up to date at:
/home/fenx/.stack/snapshots/x86_64-linux/lts-9.17/8.0.2/doc/index.html
configuring
cabal configure --builddir=/tmp/tmpThreadId11804289383846930886/dist --package-db=clear --package-db=global --package-db=/home/fenx/.stack/snapshots/x86_64-linux/lts-9.17/8.0.2/pkgdb --package-db=/home/fenx/programming/haskell/serokell/log-warper/.stack-work/install/x86_64-linux/lts-9.17/8.0.2/pkgdb -v2
/home/fenx/.stack/programs/x86_64-linux/ghc-8.2.1/bin/ghc --numeric-version
looking for tool ghc-pkg near compiler in
/home/fenx/.stack/programs/x86_64-linux/ghc-8.2.1/bin
found ghc-pkg in /home/fenx/.stack/programs/x86_64-linux/ghc-8.2.1/bin/ghc-pkg
/home/fenx/.stack/programs/x86_64-linux/ghc-8.2.1/bin/ghc-pkg --version
/home/fenx/.stack/programs/x86_64-linux/ghc-8.2.1/bin/ghc --supported-languages
/home/fenx/.stack/programs/x86_64-linux/ghc-8.2.1/bin/ghc --info
cabal: ghc-pkg dump failed
hup: 
Ran commands: 
which stack
which cabal
which haddock
which ghc
stack path --programs
which stack
appendToPath: /home/fenx/.stack/programs/x86_64-linux/ghc-8.2.1/bin
withTmpDir
rm -f /tmp/tmpThreadId11804289383846930886
mkdir /tmp/tmpThreadId11804289383846930886
echo 'haddock --hyperlinked-source'
haddock --hyperlinked-source
which haddock
echo 'build dependencies docs'
echo 'stack haddock --only-dependencies'
stack haddock --only-dependencies
which stack
stack path --snapshot-pkg-db
which stack
stack path --local-pkg-db
which stack
echo 'configuring'
echo 'cabal configure --builddir=/tmp/tmpThreadId11804289383846930886/dist --package-db=clear --package-db=global --package-db=/home/fenx/.stack/snapshots/x86_64-linux/lts-9.17/8.0.2/pkgdb --package-db=/home/fenx/programming/haskell/serokell/log-warper/.stack-work/install/x86_64-linux/lts-9.17/8.0.2/pkgdb -v2'
cabal configure --builddir=/tmp/tmpThreadId11804289383846930886/dist --package-db=clear --package-db=global --package-db=/home/fenx/.stack/snapshots/x86_64-linux/lts-9.17/8.0.2/pkgdb --package-db=/home/fenx/programming/haskell/serokell/log-warper/.stack-work/install/x86_64-linux/lts-9.17/8.0.2/pkgdb -v2
which cabal
rm -rf /tmp/tmpThreadId11804289383846930886

Exception: error running: cabal configure --builddir=/tmp/tmpThreadId11804289383846930886/dist --package-db=clear --package-db=global --package-db=/home/fenx/.stack/snapshots/x86_64-linux/lts-9.17/8.0.2/pkgdb --package-db=/home/fenx/programming/haskell/serokell/log-warper/.stack-work/install/x86_64-linux/lts-9.17/8.0.2/pkgdb -v2
exit status: 1
stderr: cabal: ghc-pkg dump failed
```

Maybe it's not a `hup` problem. I have the following `stack` version:

```
$ stack --version
Version 1.6.1, Git revision f25811329bbc40b0c21053a8160c56f923e1201b (5435 commits) x86_64 hpack-0.20.0
```

**Comment by phlummox** [comment]
This should be fixed with version 0.3.0.0 - can you try installing from the [v 0.3.0.0 source code](https://github.com/phlummox/hup/archive/v0.3.0.0.tar.gz)?
  
Using hup 0.3.0.0, `log-warper` documentation seems to build fine for me, using a docker container:

~~~
docker run --rm -it fpco/stack-build:lts-9.18 bash -c "git clone https://github.com/phlummox/hup.git && cd hup && stack install --install-ghc && stack unpack log-warper-1.8.0 &&  cd log-warper-1.8.0 && curl -O https://raw.githubusercontent.com/serokell/log-warper/master/stack.yaml && stack setup &&  hup docbuild"
~~~
 **edit:** I've now uploaded 0.3.0.0 to Hackage, so just `stack install hup` should work. Let me know if the problem persists.

**Comment by chshersh** [comment]
@phlummox Thanks for your amazing work! Everything is now working for me :slightly_smiling_face:
